packages/dev/loaders/src/glTF/2.0/Extensions/KHR_materials_transmission.ts

```diff
@@ -239,9 +239,19 @@ class TransmissionHelper {
     }
 
     /**
+     * @internal
+     * Check if the opaque render target has not been disposed and can still be used.
+     * @returns
+     */
+    public _isRenderTargetValid() {
+        return !!this._opaqueRenderTarget?._buffer;

```

Comment:
You should test `this._opaqueRenderTarget?.getInternalTexture() !== null` instead, as `_buffer` is only set when creating a texture from existing data (which is not the case here, so `_buffer = null` even if the texture is not disposed).